### PR TITLE
Fix on hover underline issue for signal icon in Tiles view

### PR DESCRIPTION
**Pull request by ahackettms**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes
In tile view, on hover it adds underline for the name field. 
However, if there is icon inside the name field, it will have underline as well. 

Fixed by adding underline only to the span elements inside Tile, and setting no underline for the parent container span of the signalField, so that only the span of the actual signal text will receive underline.
(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/@uifabric/experiments/ahackett-tilehoverfix96_2018-09-06-19-34.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Fix on hover underline issue for signal icon in Tiles view",
```
Comment (cliffkoh) [inline common/changes/@uifabric/experiments/ahackett-tilehoverfix96_2018-09-06-19-34.json]: Please add the component name in front, e.g. Tiles: Fix...

Comment (ahackettms) [reply common/changes/@uifabric/experiments/ahackett-tilehoverfix96_2018-09-06-19-34.json]: Fixed

File: packages/experiments/src/components/signals/SignalField.scss [hunk 2]
```diff
@@ -1,10 +1,10 @@
-
 @import '~office-ui-fabric-react/src/common/common';
 
 .signalField {
   &,
   &.compact {
     display: inline-flex;
+    text-decoration: none !important;
```
Comment (phkuo) [inline packages/experiments/src/components/signals/SignalField.scss]: don't use !important

**Comment by ahackettms** [comment]
@phkuo please see latest iteration where I found another approach of fix and doesn't need to use !important

**Comment by ahackettms** [comment]
Thanks

**Pull request merged**